core/src/main/java/com/linecorp/armeria/internal/common/DefaultHttpRequest.java

```diff
@@ -31,14 +31,20 @@
 
 /**
  * Default {@link HttpRequest} implementation.
+ *
+ * @deprecated Use {@link HttpRequest#streaming(RequestHeaders)} instead.
  */
+@Deprecated
```

Comment:
It is in `internal`. Do we need `@Deprecated`?

Reply:
Oops, no we don't need it. Let me revert it. 😓